app/controllers/imports_controller.rb

```
@@ -74,7 +74,7 @@ def create
     Rails.logger.error e.backtrace.join("\n")
     ExceptionReporter.call(e)
 
-    redirect_to new_import_path, alert: e.message, status: :unprocessable_entity
+    redirect_to new_import_path, alert: e.message, status: :unprocessable_content
   end
```

RuboCop (1.76.1)

[convention] Assignment Branch Condition size for `create` is too high. [<8, 37, 14> 40.36/17]

(Metrics/AbcSize)